apps/settings/js/firefox_accounts/panel.js

```diff
@@ -4,8 +4,7 @@
 'use strict';
 
 var FxaPanel = (function fxa_panel() {
-  var _ = navigator.mozL10n.get,
```

Comment:
if you are removing variable, you must remove/update it's usage, otherwise you will get exception later in code